mailbox/opensearch/src/main/java/org/apache/james/mailbox/opensearch/json/IndexableMessage.java

```diff
@@ -297,6 +297,8 @@ public EMailers getCc() {
 
     @JsonProperty(JsonMessageConstants.CC_SORT)
     public EMailers getCcSort() {
+        // workaround for ISSUE with sorts on null/empty indexed fields:
+        // https://github.com/opensearch-project/opensearch-java/issues/158
```

Comment:
Did the fix opened at https://github.com/opensearch-project/opensearch-java/pull/169 fixes this?